web/Views/Landlord/Profile.cshtml

```diff
@@ -186,4 +202,21 @@ else
             </form>
         </div>
     </div>
+</div>
+
+<!-- InviteModal -->
+<div class="modal fade" id="inviteModal" tabindex="-1" aria-labelledby="inviteModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="approveModalLabel">Landlord invite links</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                An invite link allows you to assign a landlord profile to a new user of the Bricks & Hearts site.

```

Comment:
Ampersands (and other characters) can cause issues - might want to look at this https://mrcoles.com/blog/how-use-amersands-html-encode/

Reply:
Should I change & to &amp; in the other places it appears as well? There are definitely at least a couple of other places that 'Bricks & Hearts' is written on the site, and I can find and replace them all if that will avoid bugs.